workspaces/adventure-pack/goodies/typescript/Math.lcm/index.ts

```diff
@@ -0,0 +1,21 @@
+import "../Math.gcd";
+
+declare global {
+  interface Math {
+    lcm(a: number, b: number): number;
+  }
+}
+
+Math.lcm = function (a: number, b: number): number {
+  if (a === 0 || b === 0) {
+    return 0;
+  }
+  a = Math.abs(a);
+  b = Math.abs(b);
+
+  return (a / Math.gcd(a, b)) * b;
```

Comment:
This whitespace style feels slightly more natural?

```suggestion
  }

  a = Math.abs(a);
  b = Math.abs(b);
  return (a / Math.gcd(a, b)) * b;
```